### PR TITLE
Changing Certificate type to string

### PR DESCRIPTION
**Pull request by apluchik**
Changing the certificate types to string from []byte.
related to https://github.com/microsoft/moc/pull/14

### CODE REVIEW

File: services/security/security.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ type Certificate struct {
 	// X509Thumbprint - READ-ONLY; Thumbprint of the certificate. (a URL-encoded base64 string)
 	X509Thumbprint *string `json:"x5t,omitempty"`
 	// Cer - CER contents of x509 certificate.
```
Comment (madhanrm) [inline services/security/security.go:245]: can you also update the comment saying this is base64 encoded string

File: services/security/security.go [hunk 2]
```diff
@@ -268,8 +268,8 @@ type Identity struct {
 	Type *string `json:"type,omitempty"`
 	// Token
 	Token *string `json:"token,omitempty"`
-	// Certificate
-	Certificate *[]byte `json:"certificate,omitempty"`
+	// Certificates
+	Certificate *string `json:"certificates,omitempty"`
```
Comment (madhanrm) [inline services/security/security.go:272]: is this the list of certificates that are held in map?

Comment (apluchik) [reply services/security/security.go:272]: This is the certificate passed in to the cloud agent, the map of client to cert is stored in the security.proto Identity struct.

File: services/security/security.go (hunk 1 above)
Comment (apluchik) [reply services/security/security.go]: Done.

**Review by JocelynBerrendonner — APPROVED** [review]
LGTM

**Pull request merged**